spec/models/post_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  let(:post) { Post.new(author_id: 6, title: 'test', text: 'This is for my test', likescounter: 0, commentscounter: 0) }
+
+  before { post.save }
+
+  it 'Title should not be blank' do
+    post.title = nil
+    expect(post).to_not be_valid
+  end
+
+  it 'comments counter should be integer' do
+    post.commentscounter = -1
+    expect(post).to_not be_valid
+  end
+
+  it 'title length should not be greate than 250 chars' do
+    expect(post.title.length).to be <= 250
+  end
+end
```

Comment:
- [ ] Good job adding tests that comments must be equal or greater than 0, but you must do the same thing for `like` as well
![image](https://user-images.githubusercontent.com/62705947/222687215-db2585bd-d669-4cf4-a56a-532b3c478d8e.png)